src/cockpit/389-console/src/lib/database/databaseConfig.jsx

```diff
@@ -226,6 +255,45 @@ export class GlobalDatabaseConfig extends React.Component {
         let import_cache_form;
         let db_auto_checked = false;
         let import_auto_checked = false;
+        let dblocksMonitor = "";
+        let dblocksThreshold = this.state.dblocksMonitoringThreshold;
+        let dblocksPause = this.state.dblocksMonitoringPause;
+
+        if (this.state.dblocksMonitoring) {
+            dblocksMonitor =
+                <Form isHorizontal>
+                    <FormGroup
+                        label="DB Locks Threshold Percentage"
+                        fieldId="monitoringthreshold"
+                        title="Sets the DB lock exhaustion value in percentage (valid range is 70-98). If too many locks are acquired, the server will abort the searches while the number of locks are not decreased. It helps to avoid DB corruption and long recovery. (nsslapd-db-locks-monitoring-threshold)"
+                    >
+                        <TextInput
+                            id="dblocksMonitoringThreshold"
+                            name="dblocksMonitoringThreshold"
+                            type="number"
+                            aria-describedby="dblocksMonitoringThreshold"
+                            value={dblocksThreshold}
+                            onChange={this.handleChange}
+                            validated={parseInt(dblocksThreshold) < 70 || parseInt(dblocksThreshold) > 95 ? ValidatedOptions.error : ValidatedOptions.default}
+                        />
+                    </FormGroup>
+                    <FormGroup
+                        label="DB Locks Pause Time"
+                        fieldId="monitoringpause"
+                        title="Sets the amount of time (milliseconds) that the monitoring thread spends waiting between checks. (nsslapd-db-locks-monitoring-pause)"
+                    >
+                        <TextInput
+                            id="dblocksMonitoringPause"
+                            name="dblocksMonitoringPause"
+                            type="number"
+                            aria-describedby="dblocksMonitoringPause"
+                            value={dblocksPause}
+                            onChange={this.handleChange}
+                            validated={parseInt(dblocksPause) < 1 ? ValidatedOptions.error : ValidatedOptions.default}
+                        />
```

Comment:
@mreynolds389 I've 

- made both fields **type="number"** and
- added the validation - **validated={parseInt(dblocksThreshold) < 70 || parseInt(dblocksThreshold) > 95 ? ValidatedOptions.error : ValidatedOptions.default}**.

Did I miss your point somehow?..